### ISSUE TITLE
Missing composer.json for Twig

### ISSUE THREAD

**Issue by pctechnikch** [HermanPeeren/Extengen#1]
Hello, I am following your work with great interest. Unfortunately, files are still missing and this is how this error occurs:

An error has occurred.

    0 Failed opening required 'C:\wamp64\www\dev3\libraries/yepr/vendor/autoload.php' (include_path='.;C:\php\pear')

Thanks Rene

**Comment by HermanPeeren** [comment]
Yes, sorry, momentary it is only to use Twig in the generator. Will add a composer.json in the repo and include it in the build, once a release is made.

In /libraries/yepr I have the following composer.json:
````
{
    "require": {
      "twig/twig":"^3.0"
    }
}
````
